### DISCUSSION TITLE
Can't send alerts to alertmanager due to API deprecation/removal

### DISCUSSION THREAD

**Discussion by Nickmman** [jertel/elastalert2#1406] [Q&A] [answered]
I'm using elastalert2 v2.17.0, with Alertmanager v0.27 deployed via kube-prometheus-stack.

Alerts generated with elastalert2 can't be created on alertmanager due to deprecation/removal of the v1 API. I don't see a way to specify the API version for elastalert2 to talk to alertmanager.
It's a bit odd that the v1 API doesn't work anymore when using Alertmanager v0.27 (according to the image) but it has been deprecated a long time ago regardless.

The error I get from elastalert2:
```
ERROR:elastalert:Error while running alert alertmanager: Error posting to Alertmanager: 410 Client Error: Gone for url: http://kube-prometheus-stack-alertmanager:9093/api/v1/alerts
```

When trying to curl the API endpoint of alertmanager:
```
elastalert@elastalert2-6659d49fbd-sznzz:~$ curl http://kube-prometheus-stack-alertmanager:9093/api/v1/alerts
{"status":"deprecated","error":"The Alertmanager v1 API was deprecated in version 0.16.0 and is removed as of version 0.28.0 - please use the equivalent route in the v2 API"}
elastalert@elastalert2-6659d49fbd-sznzz:~$
```

**Comment by jertel** [comment, marked as answer]
>  I don't see a way to specify the API version for elastalert2 to talk to alertmanager.

https://elastalert2.readthedocs.io/en/latest/alerts.html#alertmanager

**Reply by Nickmman** [reply]
I deployed using the helm chart and thought that it would be configured that way. Got it sorted out now, thanks!
